src/main/java/org/opensearch/eval/App.java

```diff
@@ -122,7 +122,7 @@ public static void main(String[] args) throws Exception {
             if(file.exists()) {
 
                 // Create the query set index if it does not already exist.
-                searchEngine.createQuerySetIndex();
+                searchEngine.createIndex(Constants.QUERY_SETS_INDEX_NAME, Constants.QUERY_SETS_INDEX_MAPPING);
```

Comment:
needs a check if the index already exists first. Throws an error if index already present.
Something like this:
```
if(!searchEngine.doesIndexExist(Constants.QUERY_SETS_INDEX_NAME)) {
                    searchEngine.createIndex(Constants.QUERY_SETS_INDEX_NAME, Constants.QUERY_SETS_INDEX_MAPPING);
}
```